frontend/src/pages/Dashbord.jsx

```diff
@@ -0,0 +1,3 @@
+function Dashbord() {}
```

Comment:
Dashboard, not Dashbord. anyway this component is not used : remove it (and the other unused components)